src/JanusGraph.Net/JanusGraph.Net.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.3;netstandard2.0</TargetFrameworks>
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <IncludeSource>True</IncludeSource>
+    <IncludeSymbols>True</IncludeSymbols>
+    <Version>0.1.0</Version>
+    <Title>JanusGraph.Net</Title>
+    <Authors>JanusGraph</Authors>
+    <Description>
+      JanusGraph.Net is the .NET driver of JanusGraph. It extends Apache
+      TinkerPop™'s Gremlin.Net as its core dependency with additional support
+      for JanusGraph-specific types.
+    </Description>
+    <PackageId>JanusGraph.Net</PackageId>
+    <PackageTags>janusgraph;gremlin</PackageTags>
+    <PackageProjectUrl>http://janusgraph.org/</PackageProjectUrl>
+    <PackageLicenseUrl>https://raw.githubusercontent.com/JanusGraph/janusgraph-dotnet/master/LICENSE.txt</PackageLicenseUrl>
```

Comment:
Based off this [doc](https://github.com/NuGet/Home/wiki/Packaging-License-within-the-nupkg), this could be a valid replacement to avoid the `licenseUrl` warning when the running `Pack` task:

```
<PackageLicenseExpression>Apache-2.0 AND CC-BY-4.0</PackageLicenseExpression>
```

Reply:
Thanks for digging into this, Jason.
But shouldn't `Apache-2.0` be enough for the NuGet package as the package only includes code? I would rather not add the CC-BY-4.0 license here if we don't really have to.

Reply:
I used just `Apache-2.0` for now, but tell me if you think that we also need to state `CC-BY-4.0` here and I'll change it accordingly.

Reply:
Sounds ok. Apache should be fine since the docs aren't packaged.